README.md

```diff
@@ -6,8 +6,6 @@ Initialize bitrise __config (bitrise.yml)__ based on your project.
 
 * For __Android__ projects checks for gradle files and lists all the gradle tasks, also checks for gradlew file.
 
-* For __Xamarin__ projects inspects the solution files and lists the configuration options, also checks for NuGet and Xamarin Components packages.
```

Comment:
NOTE: probably we could remove all the project type-specific content, they are outdated and some platforms are missing.

Reply:
Yeah, makes sense. I have removed the rest too.